.github/workflows/bahmni_metabase_build_publish.yaml

```diff
@@ -4,7 +4,7 @@ on:
     branches: main
```

Comment:
we can keep the name as build_publish.yml itself.

Reply:
Yes Done , Thank you.